app/index.html

```diff
@@ -122,35 +122,30 @@ <h1 class="responsive-headline">Cornell Fencing Association</h1>
                <h2 class="title-margin-bottom">News &amp; Events</h2>
                
                <div class="eight columns">
+                   
+                   <h4>Cornell Fencing Spring Banquet Weekend 2017</h4>
+                   <img style="max-width: 600px" src="images/2017AlumniTourney.jpg" alt="Alumni, Family And Current Fencers">

```

Comment:
Instead of repeating the style, please make a `class`.

Please close all HTML tags. This should be `<img />`, not `<img>`.